README.md

```diff
@@ -0,0 +1,17 @@
+Sangbogen
+========
+
+This is the current sangbog in the F-Klub
+
+Latex Environment
+-------------
+For apt systems:
+1. In a shell
+  - `git clone https://github.com/f-klubben/stregsystemet.git`
```

Comment:
This should be sangbog.git

Reply:
Ups, fixed.

I hope we will get a lot of new songs now.